docs/install.md

```diff
@@ -74,3 +87,41 @@ The different supported modes can be set by using the `hook-mode` option:
 - `podinformer`: Use a Kubernetes controller to get information about new pods. This option is racy and the first events produced by a container could be lost. This mode is selected when `auto` is used and the above modes are not available.
 - `ldpreload`: Adds an entry in `/etc/ld.so.preload` to call a custom shared library that looks for `runc` calls and dynamically adds the needed OCI hooks to the cointainer `config.json` specification. This feature only works when runc is used. Since this feature is highly experimental, it'll not be considered when `auto` is used.
 - `nri`: Use the [Node Resource Interface](https://github.com/containerd/nri). It requires containerd v1.5 and it's not considered when `auto` is used.
+
+### Specific Information for Different Platforms
+
+This section explains the additional steps that are required to run Inspektor
+Gadget in some platforms.
+
+#### Minikube
+
+You should create the minikube cluster in different ways according to the gadget
```

Comment:
I think you mean:
Depending on how you deployed the Minikube cluster, you'll be able to use different gadgets.

If you're running Minikube inside a VM (for example because you're running it on a cloud provider or you're using a Windows machine with a Linux VM inside), you don't really choose how to create the cluster.

Reply:
>  you don't really choose how to create the cluster.

I think in the case of minikube the users control how to create the cluster using minikube. Don't they?

Reply:
Well, if they don't have access to a bare-metal linux machine, they can't choose the native option, that's what I'm saying.

Reply:
I think minikube is mainly used for testing and learning purposes and the cluster is usually created on the user's computer. That's why I think the user has control on how the **minikube** cluster is created.